libconcurrent/concurrent/hsynch.c

```diff
@@ -106,8 +113,8 @@ void HSynchStructInit(HSynchStruct *l, uint32_t nthreads, uint32_t numa_regions)
     if (numa_regions == HSYNCH_DEFAULT_NUMA_POLICY) {
         l->numa_policy = true;
 #ifdef NUMA_SUPPORT
```

Comment:
A few comments would help reviewing the purpose of this change and make the code easier to follow.

Reply:
Done.